scripts/managePOIs.js

```diff
@@ -0,0 +1,110 @@
+const express = require('express');
+const server = express();
+const path = require('path');
+const { PATH_SCRAPED_ARTWORKS, PATH_MANAGED_POIs } = require('../config/file-paths');
+
+const Artwork = require('../lib/artwork.js');
+const POIManager = require('../lib/POIManager.js')
+
+// config vars
+const PORT = 3000;
+
+server.use(express.json());
+
+// POI Manager
+const app = new POIManager();
+app.refreshArtworks();
+
+
+//app.writeArtworkToFile();
+
+// const sampleArtwork = require(PATH_SCRAPED_ARTWORKS)[7];
+// console.log(sampleArtwork)
+// app.writeArtworkToFile(sampleArtwork);
```

Comment:
remove? Or put this in a function that's only called when a debug flag is set?

Reply:
got it. will remove this